api/spec/v1.yaml

```diff
@@ -1370,6 +1398,83 @@ components:
       description: |
         A ParaTime block.
 
+    RuntimeTransactionList:
+      type: object
+      properties:
+        transactions:
+          type: array
+          items:
+            $ref: '#/components/schemas/RuntimeTransaction'
+      description: |
+        A list of consensus transactions.
+
+    RuntimeTransaction:
+      type: object
+      properties:
+        round:
+          type: integer
+          format: int64
+          description: The block round at which this transaction was executed.
+          example: 3379702
+        hash:
+          type: string
+          description: The Oasis cryptographic hash of this transaction's encoding.
+          example: 8394f682150e5f62b02f197d16b4769d032cb1c1b7a6dcf853ba1b12626e080b
+        eth_hash:
+          type: string
+          description: |
+            The Ethereum cryptographic hash of this transaction's encoding.
+            Absent for non-Ethereum-format transactions.
+          example: 9e6a5837c6366d4a7e477c71ffe32d40915cdef7ef209792259e5ee70caf2705
+        sender_0:
+          type: string
+          description: The Oasis address of this transaction's 0th signer.
+          example: oasis1qz670t637yyxshnlxhjj5074wgwl94d0x5x69zqd
+        nonce_0:
+          type: integer
+          format: int64
+          description: The nonce used with this transaction's 0th signer, to prevent replay.
+          example: 114194
+        fee_amount:
+          type: string
+          description: |
+            The fee that this transaction's sender committed to pay to execute
+            it (total, native denomination, ParaTime base units, as a string).
+          example: "3000000000000000"
+        fee_gas:
+          type: integer
+          format: int64
+          description: |
+            The maximum gas that this transaction's sender committed to use to
+            execute it.
+          example: 30000
+        method:
+          type: string
+          description: The method that was called.
+          example: "evm.Call"
+        body:
+          type: string
+          description: The method call body.
+          example: "o2RkYXRhQGV2YWx1ZUgBY0V4wN0lc2dhZGRyZXNzVDlzcMsG8d3D2LY5FTn+1svi/l+V"
+        to:
+          type: string
+          description: |
+            A reasonable "to" Oasis address associated with this transaction,
+            if applicable. The meaning varies based on the transaction method.
```

Comment:
```suggestion
            if applicable. The meaning varies based on the transaction method. Some notable examples:
              - For `method = "consensus.Transfer"`, this is the paratime account receiving the funds.
              - For `method = "consensus.Deposit"`, this is the paratime account receiving the funds.
              - For `method = "consensus.Withdraw"`, this is a consensus (!) account receiving the funds.
              - For `method = "evm.Create"`, this is the address of the newly created smart contract.
              - For `method = "evm.Call"`, this is the address of the called smart contract
```

Reply:
@pro-wh Did these explanations never get merged in, or was it subsequently removed? @lukaw3d was looking at them here, but I can't find them in `main`,

I should also add that these are my best-effort interpretations of what `.To` contains, please review them.

Reply:
probably force-pushed away by accident